### DISCUSSION TITLE
How can I update specific fields for existing devices using a CSV file

### DISCUSSION THREAD

**Discussion by aaanserali** [netbox-community/netbox#14256] [General]
I am unable to update specific fields in the existing devices through a CSV file. I have more than 500 devices, and I recently created custom fields. I need to add values for all the servers through CSV. I have tried updating, but it is not working properly. Is there any procedure to accomplish this smoothly?

**Comment by candlerb** [comment]
I expect you have an error in your CSV, but since you haven't shown it, we can't tell.

You also haven't said what version of Netbox you're using. Until recently, Netbox ignored CSV columns with invalid headings. So if you get the heading wrong, nothing will happen (i.e. no error is shown, but no update is made). See #11617
